.idea/workspace.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
-    <list default="true" id="6512ec66-1172-4fa5-996a-1c8e56f3767e" name="Default" comment="" />
+    <list default="true" id="6512ec66-1172-4fa5-996a-1c8e56f3767e" name="Default" comment="">
+      <change beforePath="$PROJECT_DIR$/.idea/workspace.xml" afterPath="$PROJECT_DIR$/.idea/workspace.xml" />
+    </list>
```

Comment:
Were all these xml changes brought in automatically with the packages that you used for the toggle menu?

Reply:
^ same